cob_monitoring/src/cpu_monitor.py

```diff
@@ -89,7 +90,7 @@ def check_core_temps(self, interval=1):
         diag_level = DiagnosticStatus.OK
 
         try:
-            netdata_core_temp, error = query_netdata('sensors.coretemp_isa_0000_temperature', interval)
+            netdata_core_temp, error = query_netdata(self._netdata_module_name_core_temps, interval)
```

Comment:
can we not try both names and use the one that works...and only DiagnosticStatus.ERROR if BOTH fail
would be easier than using a parameter